packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1364,10 +1422,12 @@ class _NullTableColumnWidth extends TableColumnWidth {
   const _NullTableColumnWidth();
 
   @override
-  double maxIntrinsicWidth(Iterable<RenderBox> cells, double containerWidth) => throw UnimplementedError();
+  double maxIntrinsicWidth(Iterable<RenderBox> cells, double containerWidth) =>
+      throw UnimplementedError();
 
   @override
-  double minIntrinsicWidth(Iterable<RenderBox> cells, double containerWidth) => throw UnimplementedError();
+  double minIntrinsicWidth(Iterable<RenderBox> cells, double containerWidth) =>
+      throw UnimplementedError();
```

Comment:
Looks like you've made some unintended format changes. 

Please read https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting


Please revert the format changes

Reply:
Fixed this